app/workers/check_for_new_commits_on_github_worker.rb

```diff
@@ -4,6 +4,8 @@
 require_relative "../shared/models/job_trigger"
 require_relative "../shared/logging_module"
 
+require "time"
```

Comment:
Sounds like this requires some time 

🦆s after adding this 👨 joke

Reply:
![shockedguy](https://user-images.githubusercontent.com/1304821/38073016-b858f5d0-335b-11e8-9457-715bb10f9fe9.gif)